server/schedulers/evict_slow_store.go

```diff
@@ -89,6 +89,20 @@ func (conf *evictSlowStoreSchedulerConfig) getSchedulerName() string {
 	return EvictSlowStoreName
 }
 
+func (conf *evictSlowStoreSchedulerConfig) getStores() []uint64 {
+	// TODO: shoud we add a lock?
```

Comment:
maybe?

Reply:
seem no need

Reply:
yes, we do not need it now, but with the design, we need it in the future.